src/util.nim

```diff
@@ -628,10 +628,10 @@ proc getOrDefault*[T](self: openArray[T], i: int, default: T): T =
 
 proc getRelativePathBetween*(fromPath: string, toPath: string) : string =
   ## Given the `fromPath`, usually the project root, return the relative
-  ## path of the file's `toPath`. Return nothing if its outside the project root
-  ## or if `toPath` is an empty string.
+  ## path of the file's `toPath`. Return nothing if its outside the project root,
+  ## if `toPath` is an empty string or, if Dockerfile contents was passed via stdin.
   result = toPath.relativePath(fromPath)
-  if result.startsWith("..") or result == "":
+  if result.startsWith("..") or result == "" or result.endsWith(":stdin:"):
```

Comment:
👍 

should we make the `:stdin:` string a constant somewhere so that its referred in the same way everywhere?

Reply:
good point